client/packages/config/src/routes.ts

```diff
@@ -65,5 +65,23 @@ export enum AppRoute {
 }
 
 export enum ExternalURL {
-  PublicDocs = 'https://docs.msupply.foundation/docs',
+  // PublicDocs = 'https://docs.msupply.foundation/docs',
+  PublicDocs,
 }
+
+export interface LocalisedExternalUrlProps {
+  url: ExternalURL;
+  locale: String | undefined;
+}
+
+export const localisedExternalUrl = ({
+  url,
+  locale,
+}: LocalisedExternalUrlProps) => {
+  switch (url) {
+    case ExternalURL.PublicDocs: {
+      const localeUrlInsert = locale == 'en' ? '' : `${locale}/`;
```

Comment:
as per the issue - we can't do this! there aren't docs for all languages (yet)
we only have French and some Portuguese, so you need to check the locale